go/vt/dbconfigs/dbconfigs.go

```diff
@@ -152,6 +156,7 @@ func registerPerUserFlags(userKey string, uc *UserConfig, cp *mysql.ConnParams)
 	flag.StringVar(&cp.UnixSocket, "db-config-"+userKey+"-unixsocket", "", "deprecated: use db_socket")
 	flag.StringVar(&cp.Charset, "db-config-"+userKey+"-charset", "utf8", "deprecated: use db_charset")
 	flag.Uint64Var(&cp.Flags, "db-config-"+userKey+"-flags", 0, "deprecated: use db_flags")
+	flag.Var(&cp.SslMode, "db-config-"+userKey+"-ssl-mode", "deprecated: use db_ssl_mode")
```

Comment:
@deepthi I can also drop it, but I kept it for consistency. If it's preferred, I can remove this specific one :smile:.

Reply:
I think this implies a potentially bigger thing though maybe for a separate PR, which is that the `UseSSL` flag for a specific user config might also need to be extended? 

And I think that then the client certificate / private key flags that exist in the global config now probably more belong to the specific user config? 

That all seems like a bigger / separate change though then?

Reply:
I removed this legacy flag, since there is I think a question around more specific per user SSL config settings, but that can be separate from these changes.